### DISCUSSION TITLE
How to fine-tuned paddle ocr to specific language like indonesian

### DISCUSSION THREAD

**Discussion by blizaga** [PaddlePaddle/PaddleOCR#14114] [Q&A]
I managed to fine tune the paddleocr model but I used yaml configuration for English, what I want is paddle ocr can be multilingual, namely Indonesian with datasets that I collect myself. 

And when I do inference, paddleocr only detects that it is English not Indonesian.

**Comment by GreatV** [comment]
please refer to: https://paddlepaddle.github.io/PaddleOCR/latest/ppocr/model_train/training.html

**Reply by blizaga** [reply]
I am still confused from the documentation, which part refers so that I can train paddleocr in Indonesian?
